services/py-api/src/server/routes/participants_routes.py

```diff
@@ -30,16 +31,22 @@ def _tx_manager(db_manager: DB_MANAGER) -> TransactionManager:
     return TransactionManager(db_manager)
 
 
-def _service(
+def _h_service(
     p_repo: ParticipantsRepository = Depends(_p_repo),
     t_repo: TeamsRepository = Depends(_t_repo),
     tx_manager: TransactionManager = Depends(_tx_manager),
-) -> ParticipantService:
-    return ParticipantService(p_repo, t_repo, tx_manager)
+) -> HackathonService:
+    return HackathonService(p_repo, t_repo, tx_manager)
```

Comment:
Question: Does this make sure that we only have an instance of each Class running? What is the purpose?

Reply:
Yeah its fine! Thank you!